docs/JoshuaSCochrane.md

```diff
@@ -0,0 +1,7 @@
+## Josh Cochrane
+
+Hometown: Watertown, WI
```

Comment:
@JoshuaSCochrane  Could you be more specific on your hometown... maybe your addy, blood type, first born, mother's maiden name, last for of your SSN, etc, etc